### DISCUSSION TITLE
自己创建的prompt是否可以支持编辑，这样可以更好的调优

### DISCUSSION THREAD

**Discussion by antilmid** [timqian/openprompt.co#36] [Ideas]
自己创建的prompt是否可以支持编辑，这样可以更好的调优

**Comment by timqian** [comment]
<img width="436" alt="Screen Shot 2023-04-04 at 14 34 58" src="https://user-images.githubusercontent.com/5512552/229707899-75a501d6-5506-4bfd-a8de-4088f80e5ed2.png">
如果是你创建的 prompt, prompt 页面会有编辑和删除按钮

**Reply by antilmid** [reply]
谢谢
